src/index.ts

```diff
@@ -4,8 +4,11 @@ import { parsers as htmlParsers } from 'prettier/parser-html';
 import { parsers as typescriptParsers } from 'prettier/parser-typescript';
 
 import { defaultPreprocessor } from './preprocessors/default-processor';
+import { sveltePreprocessor } from './preprocessors/svelte-preprocessor';
 import { vuePreprocessor } from './preprocessors/vue-preprocessor';
 
+const { parsers: svelteParsers } = require('prettier-plugin-svelte');
```

Comment:
`prettier-plugin-svelte@2` works with `prettier@2` and has no types. This `require` is used to skip typing the module.
`prettier-plugin-svelte@3` has typings.